spec/models/task_spec.rb

```diff
@@ -1,5 +1,53 @@
 require 'rails_helper'
 
 RSpec.describe Task, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  describe 'validation' do
+
+    context '値を入れなくてはならないカラムのバリデーションが有効であるか確認する' do
+      # すべての属性の値があれば有効
+      it 'is valid with all attributes' do
+        task = build(:task)
+        expect(task).to be_valid
+        # エラーは空である
```

Comment:
表現が分かりづらいと思ったので、修正した方が良いかと思いました。
```suggestion
        # バリデーション判定後のインスタンスの持つエラーメッセージの配列は空である
```